doc/manual.rst

```diff
@@ -6433,38 +6433,39 @@ generates:
     unsigned int flag:1;
   };
 
-Alignas pragma
---------------
 
-The ``alignas`` pragma is for variables and object field members. It
-modifies the alignment requirement of the thing being declared. The
-argument must be a constant power of 2 or 0.  Valid non-zero
-alignments that are weaker than another alignas pragmas on the same
-declaration are ignored.  Alignments that are weaker that the
-alignment requirement of the type are ignored. ``alignas(0)`` is
-always ignored.
+Align pragma
+------------
+
+The `align`:idx: pragma is for variables and object field members. It
+modifies the alignment requirement of the entity being declared. The
+argument must be a constant power of 2. Valid non-zero
+alignments that are weaker than nother align pragmas on the same
+declaration are ignored. Alignments that are weaker that the
+alignment requirement of the type are ignored.
```

Comment:
Why did you remove the mentioning of the `0` argument? `0` is not a power of two nor a valid alignment, but the behavior is well specified as ignored. There is logic for that and there is a test case for that.

Reply:
I know, I'll remove it, what's the point of it? Copying bad design is not good design.

Reply:
@cooldome @mratsim Since you reviewed my original PR, I would like to have your opinion about this.